src/components/fields/text-field/types.ts

```diff
@@ -2,15 +2,24 @@ import { FormInputProps } from "@lifesg/react-design-system/form/types";
 import { TComponentOmitProps } from "../../frontend-engine";
 import { IBaseFieldSchema } from "../types";
 
+export enum ETextTransform {
+	UPPERCASE = "uppercase",
+}
+
 type TCustomOptions = {
 	preventCopyAndPaste?: boolean | undefined;
 	preventDragAndDrop?: boolean | undefined;
+	textTransform?: ETextTransform | undefined;
```

Comment:
instead of enum, can we have a union of the possible values? the reason is the devs will not have access to this enum when defining the schema

```suggestion
	textTransform?: "uppercase" | undefined;
```

Reply:
ooo so thats why enum is rarely used from certain things ok